### ISSUE TITLE
Add CMS story

### ISSUE THREAD

**Issue by jonathanirvings** [ioi-2022/cms#12]
CMS should encourage more socialising in IOIs by allowing contestants to view other contestants' stories

![Screenshot 2021-02-28 at 2 05 32 AM](https://user-images.githubusercontent.com/6689060/109395884-b29b9600-7969-11eb-968d-64365e337934.png)

**Comment by spdskatr** [comment]
based

**Comment by fushar** [comment]
Yeah, this could solve the problem with online IOI, where you cannot observe the reactions of contestants in front of / besides your desk, like the other normal onsite IOIs.

With this feature, for each contestant, we can randomly show 3 other "stories" of other contestants, to simulate the above experience.
